roles/ipxe/tasks/main.yml

```diff
@@ -58,12 +61,12 @@
     chdir: "{{ ansible_env.HOME }}/ipxe/src"
```

Comment:
@benbp do you know if we did these ipxe builds in the users home dir due to some permissions thing? or any other issue that may not be obvious to me? If not would you like me to generate a new story to implement the suggestion above? @stuart-stanley thanks for the feed back!